src/renderer/components/ClearCacheButton.tsx

```diff
@@ -1,18 +1,29 @@
 import React, { useState } from "react";
-import { ipcRenderer } from "electron";
+import { ipcRenderer, remote } from "electron";
 import { Button, ButtonProps } from "@material-ui/core";
+import { t } from "@transifex/native";
 import DeleteIcon from "@material-ui/icons/Delete";
 import CheckCircleIcon from "@material-ui/icons/CheckCircle";
 
+const { dialog } = remote;
+const message = t("All block data will be deleted and re-downloaded again. This process can take up to 30 minutes. Are you sure to continue?");
+
 interface IClearCacheButtonProps extends ButtonProps {
   disabled?: boolean;
 }
 
 const ClearCacheButton = (props: IClearCacheButtonProps) => {
   const [isCleared, setClearState] = useState(false);
 
-  const handleClick = () => {
-    if (window.confirm("Are you sure you want to clear cache?")) {
+  const handleClick = async () => {
+    const result = await dialog.showMessageBox({
+      type: "question",
+      buttons: ["Yes", "No"],
```

Comment:
It would be better if we also translate these button labels too.

Reply:
It seems like the Windows (or Electron, unsure) automatically translates common button labels like this one. It can be seen in the screenshot above.